### PR TITLE
feat: Remove google font

### PR DESCRIPTION
**Pull request by joschahenningsen**
Hi there! We're using a slightly modified version of your plugin to avoid requests to fonts.googleapis.com. Videojs already contains the same icon in their font. If you want, feel free to apply theese changes upstream :relaxed: 
(unsure if this would break compatibility with earlier versions of vjs though)

Demo: https://live.mm.rbg.tum.de/w/test/26

### CODE REVIEW

**Comment by mister-ben** [comment]
That isn't a like for like replacement as the current CSS uses the icons with numbers on them. It could be useful to have this as an alternative though, loading a whole font for a couple of icons isn't ideal.

**Comment by mister-ben** [comment]
On reflection the numbers on the icon are so small that they're not really worth it, so I'll merge this. Thanks for the contribution!

**Comment by joschahenningsen** [comment]
Sweet, do you want me to remove the extra css classes (skip-30, skip-10, skip-15 etc)?

**Comment by mister-ben** [comment]
Yes please. I was also wondering about rotating the icons 45° so there's no confusion with the same icon used for "replay".

**Comment by joschahenningsen** [comment]
> I was also wondering about rotating the icons 45° so there's no confusion with the same icon used for "replay".

How does this look?
image

File: src/plugin.css [hunk 1]
```diff
@@ -1,77 +1,45 @@
...
-      content: '\e059';
-    }
-    &.skip-back.skip-5::before {
-      content: '\e05b';
-    }
-    &.skip-back.skip-30::before {
-      content: '\e05a';
+      transform: rotate(-45deg);
+      -ms-transform: rotate(-45deg); /* IE 9 */
+      -webkit-transform: rotate(-45deg); /* Safari */
```
Comment (joschahenningsen) [inline src/plugin.css:15]: Unfortunately I can't  test this on Safari or IE.

Comment (mister-ben) [reply src/plugin.css:15]: IE9 and Safari 8 are beyond ancient now anyway

**Comment by mister-ben** [comment]
Thanks a lot. The rotated icons look good.
About removing the css -  I thought you meant only to remove the now redundant rules for those in the CSS. I'd actually still add classes to the elements in case anyone is customising the icons and relying on those. I'll revert that change separately. Sorry for the confusion, and thanks again.

**Comment by joschahenningsen** [comment]
You're welcome. Also thanks for your contributions to videojs and its plugins, they save my butt from time to time :D

**Pull request merged**